src/stores/careerPlanningAnswersStore.test.ts

```diff
@@ -130,6 +130,7 @@ describe('useCareerPlanningAnswersStore', () => {
     const encodedLink = store.getEncodedData();
 
     // Check that the encoded link is a valid base64 string
+    // eslint-disable-next-line sonarjs/slow-regex
     expect(encodedLink).toMatch(/([A-Za-z0-9+/=]+)$/);
```

Comment:
Maybe this is okay like this as this is only a **test** and not used in actual code. Or can removed when better regex is provided.